src/js/index.js

```diff
@@ -1,7 +1,74 @@
+const baseUrl = "https://js-todo-list-9ca3a.df.r.appspot.com";
+
+
 const onUserCreateHandler = () => {
-  const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+    const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+    if (userName == null || userName == undefined)
+        return;
+    if (userName.length < 2)
+        alert("사용자 이름은 최소 2글자 이상입니다.");
+
+    // 사용자 추가하기
+    $.ajax({
+        type: "POST",
+        url: `${baseUrl}/api/users`,
+        data: { "name": userName },
+        success: function(result) {
+            getAllUser(result._id);
+        }
+    });
 }
 
-const userCreateButton = document.querySelector('.user-create-button')
-userCreateButton.addEventListener('click', onUserCreateHandler)
+//USERlIST 호출
+function getAllUser(id) {
+    $.ajax({
+        type: "GET",
+        url: `${baseUrl}/api/users`,
+        success: function(result) {
+            createUserBtn(result, id);
+        }
+    })
+}
+
+//버튼생성
+function createUserBtn(userListArr, focusId) {
+    let userListDiv = document.getElementById("user-list");
```

Comment:
```suggestion
    let $userListDiv = document.getElementById("user-list");
```
저는 일반 변수와 Element를 담은 변수를 구분하기 위해 '$변수명' 방식 사용하는데, 구분할 때 편리합니다. 😊